uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -627,7 +648,7 @@ INPUTS:
   - $yspace         : y-space of the image
   - $zspace         : z-space of the image
   - $time           : time dimension of the scan
-  - $seriesUID      : seriesUID of the scan
+  - $seriesUID      : C<SeriesUID> of the scan
```

Comment:
why put this in C<> but nothing else before? like x-space, y-space, etc...

Reply:
good question. Changed it :)